resources/lang/pirate/mainLang.php

```diff
@@ -537,6 +537,11 @@
     'privacyPolicy'     => 'Datenschutz',
     'impressum'         => 'Impressum',
 
+    'agreeWithPrivacy'  => 'Bitte scroll nach unten und stimm der Datenschutzrichtlinie von Lara zu, um die Website zu nutzen',
```

Comment:
Will need to formulate this differently, we can't tell our users to "scroll past this", we should tell them they "need to read it", at least the short summary provided in next tabs ;)